compiler-libs-406/parmatch.ml

```diff
@@ -376,7 +376,7 @@ let is_cons = function
 
 let pretty_const c = match c with
 | Const_int i -> Printf.sprintf "%d" i
-| Const_char c -> Printf.sprintf "%C" c
+| Const_char i -> Printf.sprintf "%C" (Char.unsafe_chr i)
 | Const_string (s, _) -> Printf.sprintf "%S" s
```

Comment:
This function should be adapted, since our char could be a large value. The '%c' may not be appropriate

Reply:
Sorry, I got stuck here, do you have any suggestion to safely print `int` as `char`?